### DISCUSSION TITLE
ScrapingAnt (free)

### DISCUSSION THREAD

**Discussion by Dan4ic** [towfiqi/serpbear#64] [General]
Hi,
Is the api from ScrapingAnt (free) no longer suitable?
It displays a mark near the keyword:
"subscribe to the paid plan for using it"

https://skr.sh/sHvqq0Qgtgf

**Comment by towfiqi** [comment]
Probably. I had the same issue and switched to ScrapingRobot and everything seems to work.
Strangely, on their website, they are still advertising the free plan. Could be an issue on thier end. Need to contact them and let them know about it.

**Reply by Dan4ic** [reply]
> Probably. I had the same issue and switched to ScrapingRobot and everything seems to work. Strangely, on their website, they are still advertising the free plan. Could be an issue on thier end. Need to contact them and let them know about it.

Yes, apparently they have a problem ... contacted their support, waiting for an answer ...)

**Comment by Dan4ic** [comment]
Got an answer from support ScrapingAnt:

`We don't allow several websites to be scraped at Free plan as users are tend to abuse free plan with multiple accounts to take advantage out of it.`

Although I have one site connected, but apparently it does not matter
